nibabel/orientations.py

```diff
@@ -349,6 +349,7 @@ def axcodes2ornt(axcodes, labels=None):
            [ 0., -1.],
            [ 2.,  1.]])
     """
+    axcodes = axcodes.upper()
```

Comment:
According to the docsring, `axcodes` is a tuple not a str. So, `.upper()` does not exist.

Reply:
Thanks; I hadn't noticed that : ) It fixed my loading issue because the .trk files I'm trying to load have orientation stored as a string, but that does break on tuples. I exchanged that line for a loop that generates an upper-case tuple from either a string or a tuple (see below). How's this?

```
upper_str = []
    for c in axcodes:
        upper_str.append(c.upper())
    axcodes = tuple(upper_str)
```

Reply:
How about the following?

```Python
axcodes = tuple(c.upper() for c in axcodes)
```

Reply:
Thanks, @effigies